src/test/rmc/ArithOperators/main.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-pub fn main() {
+fn main() {
```

Comment:
Isn't this change needed for each test as in #523?

Reply:
No. It's not needed as in #523. With this change RMC will accept both public and non public main.

Reply:
Yes, I understand it is not strictly need but nice to have. Anyway, let me take care of that in #659

Reply:
Thanks